include/z64cutscene_commands.h

```diff
@@ -11,445 +11,266 @@
- * FORMAT
- *   Capital U is Unused , endFrame duplicate
- *   0000002D 00000001 ttttssss eeeeUUUU
- *   size = 0x10
+ * Controls various types of screen transitions.
+ * @see `CutsceneTransitionTypes` for type options.
  */
-#define CS_SCENE_TRANS_FX(transitionType, startFrame, endFrame) \
-    CS_CMD_SCENE_TRANS_FX, 0x00000001, CMD_HH(transitionType, startFrame), CMD_HH(endFrame, endFrame)
+#define CS_TRANSITION(type, startFrame, endFrame) \
+    CS_CMD_TRANSITION, 0x00000001, CMD_HH(type, startFrame), CMD_HH(endFrame, endFrame)
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000056 eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_START_SEQ` entries.
  */
-#define CS_PLAY_BGM_LIST(entries) CS_CMD_PLAYBGM, CMD_W(entries)
+#define CS_START_SEQ_LIST(entries) \
+    CS_CMD_START_SEQ, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 sequence (q), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   qqqqssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Starts a sequence at the specified time.
+ * @note The sequence ID is subtracted by 1 before being used. Add +1 to the desired sequence ID when passing it in.
+ * @note The value for `endFrame` is not used, only the starting frame matters.
  */
-#define CS_PLAY_BGM(sequence, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(sequence, startFrame), CMD_HH(endFrame, unused0), \
+#define CS_START_SEQ(seqId, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
+    CMD_HH(seqId, startFrame), CMD_HH(endFrame, unused0), \
     CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
     CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000057 eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_STOP_SEQ` entries.
  */
-#define CS_STOP_BGM_LIST(entries) CS_CMD_STOPBGM, CMD_W(entries)
+#define CS_STOP_SEQ_LIST(entries) \
+    CS_CMD_STOP_SEQ, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 sequence (q), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   uuqqssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Stops a sequence at the specified time.
+ * @note The sequence ID is subtracted by 1 before being used. Add +1 to the desired sequence ID when passing it in.
+ * @note The value for `endFrame` is not used, only the starting frame matters.
  */
-#define CS_STOP_BGM(sequence, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(sequence, startFrame), CMD_HH(endFrame, unused0), \
+#define CS_STOP_SEQ(seqId, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
```

Comment:
So seqIdPlusOne ?

Reply:
yea. Though I personally dont like indicating plusOne or minusOne in variable names. It just makes it more noisy and detracts from the point of it being a seq id. It feels more relevant for a documentation comment to me.
Probably not a popular opinion tho

Reply:
Yeah I myself like having Plus/MinusOne in names, it looks less weird then in code `seqIdPlusOne - 1` than `seqId - 1`
Also I think some audio code has a bunch of seqIdPlusOne s already (iirc) so it wouldn't be a new thing